packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -349,8 +385,21 @@ abstract class PrimaryPointerGestureRecognizer extends OneSequenceGestureRecogni
   }
 
   @override
-  void rejectGesture(int pointer) {
+  void acceptGesture(int pointer) {
+    // Ignore state 'ready' here because that would happen if this recognizer
+    // won by a sweep.
     if (pointer == primaryPointer && state == GestureRecognizerState.possible) {
+      state = GestureRecognizerState.accepted;
+    }
+  }
+
+  @override
+  void rejectGesture(int pointer) {
+    // Ignore state ready here because that would happen if this recognizer won
+    // by a sweep.
```

Comment:
why would we end up here if we win? i'm confused

Reply:
I'm afraid I may have cooked an unsavory dish of copypasta. Removed.